<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactDock</class>
 <widget class="QWidget" name="ContactDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dock</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="FilterLineEdit" name="m_pFilterLE">
     <property name="placeholderText">
      <string>Filter</string>
     </property>
     <property name="showClearButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="ContactView" name="m_pView">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="indentation">
       <number>5</number>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <attribute name="headerVisible">
       <bool>true</bool>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="AutoComboBox" name="m_pSortByCBB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLength</enum>
     </property>
     <property name="minimumContentsLength">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ContactView</class>
   <extends>QTreeView</extends>
   <header>widgets/contactview.h</header>
  </customwidget>
  <customwidget>
   <class>FilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/filterlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>AutoComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/autocombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
